src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -73,6 +108,8 @@
       <PreviouslySourceBuiltNuGetSourceName>previously-source-built</PreviouslySourceBuiltNuGetSourceName>
       <ReferencePackagesNuGetSourceName>reference-packages</ReferencePackagesNuGetSourceName>
       <SourceBuildSources>$(SourceBuildSources);$(PrebuiltNuGetSourceName);$(PreviouslySourceBuiltNuGetSourceName);$(ReferencePackagesNuGetSourceName)</SourceBuildSources>
+      <BuildCommand>$(BuildCommand) /p:RestoreConfigFile=@(NuGetConfigFiles)</BuildCommand>
```

Comment:
RestoreConfigFile seems like too generic a name. Perhaps `NuGetRestoreConfigFile`?

Reply:
That's the official nuget property...